src/Converter.php

```diff
@@ -242,9 +243,13 @@ public function convert(Project $project, File $file): string
                             $parameter = $this->getParameter($project, $objectType, $namespace, $object, $function, $text);
 
                             if ($parameter) {
-                                $output .= $parameter[0].' ';
-
                                 $outputDefaultValue = (bool) $parameter[1];
+                                if ($outputDefaultValue && $nullableTypes) {
+                                    $output .= '?';
+                                    $outputDefaultValue = false;
```

Comment:
The default value can be something else than `null` (eg: `?string = 'bar'`).

Reply:
I'm afraid I don't get what you're saying. `$outputDefaultValue` seems to only trigger the addition of ` = null`. When a nullable parameter already has a default value, the latter is not changed by the converter.